packages/styles/components/Tooltip.module.scss

```diff
@@ -76,23 +78,24 @@ $arrow-size: 10px;
           left: 42%;
           position: absolute;
           top: 127%;
```

Comment:
To avoid side effects is a good idea rewrite it avoiding to using magic numbers 😊

Reply:
Done 🥰